source/dbo/Stored Procedures/Reusable Notification/uspGenerateReusableNotificationTreatmentRegimen_ETS.sql

```diff
@@ -17,15 +17,15 @@ CREATE PROCEDURE [dbo].[uspGenerateReusableNotificationTreatmentRegimen_ETS]
 AS
 
 	UPDATE dbo.ReusableNotification_ETS
-		SET TreatmentRegimen = 'StandardTherapy'
+		SET TreatmentRegimen = 'Standard therapy'
```

Comment:
I would use the lookup table here as well, just so it is a little bit more maintainable, even if all you do is declare 3 variables at the beginning, one containing one SELECT from the lookup table.

Reply:
Yeah I was mainly following the `OutcomeLookup` table's pattern but I think both view or table would have worked fine. I've gone for the local variable approach as you suggested